### ISSUE TITLE
DWC not continuously updating anymore

### ISSUE THREAD

**Issue by wilriker** [chrishamm/DuetSoftwareFramework#27]
`DCS`: 1.0.2.1
`DWS`: 1.0.1.1
`DWC`: 2.0.0-RC8-b1
`RRF`: `1c85904a`

Since updating to the latest DuetSoftwareFramework and RRF `DWC` no longer continuously updates values like temperatures, current VIN value, DHT sensor values. Sometimes it also does not populate the tools section of DWC. This is with the printer being idle. I did not yet test while printing.

EDIT: This also leads to ATX Power Control Panel being hidden until `M80` (or I assume also `M81`) is sent manually.

**Comment by beikeland** [comment]
Confirmed, hovering over the Vin value does show a tool tip with min/max so there is at least three datapoints.
![image](https://user-images.githubusercontent.com/2701678/61597966-70e82c80-ac17-11e9-8849-cbd001218774.png)

**Comment by wilriker** [comment]
Temperatures are not updated while heating (when temp set manually via DWC).

**Comment by wilriker** [comment]
Fixed in `DCS` 1.0.2.2.

**Comment by wilriker** [comment]
Reopening since this is again the case with `DCS` 1.0.3.0.

**Comment by wilriker** [comment]
Now here is something really strange: If `DCS` gets killed by an error like described in #34 then after it runs again the updates are working and the WebSocket will see continuous updates. This also works across page reloads.

Restarting `DCS` with a working `config.g` does not allow it to work. 

I tracked it down to
```
; Tools
M563 P0 S"Hotend" D0 H1                            ; Define tool 0
```

If that's in `config.g` then no updates will flow. If I comment it out status updates work. So most likely this is tied to #47.

**Comment by chrishamm** [comment]
Fixed in DCS 1.0.3.1.
